Services/source/general.c

```diff
@@ -136,14 +137,32 @@ SAT_returnState general_app(csp_packet_t *packet) {
         }
         memcpy(&packet->data[STATUS_BYTE], &status, sizeof(int8_t));
         set_packet_length(packet, sizeof(int8_t) + 1); // +1 for subservice
-        csp_send(conn, packet, 50);
+        csp_send(conn, packet, CSP_TIMEOUT);
 
         if (status == 0) {
             reboot_system(reboot_type);
         }
 
         break;
 
+    case GET_UHF_WATCHDOG_TIMEOUT:
+        status = 0;
+        memcpy(&packet->data[STATUS_BYTE], &status, sizeof(int8_t));
+        unsigned int timeout = get_uhf_watchdog_delay();
+        memcpy(&packet->data[OUT_DATA_BYTE], &timeout, sizeof(unsigned int));
+        set_packet_length(packet, sizeof(int8_t) + sizeof(unsigned int) + 1); // +1 for subservice
+        csp_send(conn, packet, CSP_TIMEOUT);
+        break;
+
+    case SET_UHF_WATCHDOG_TIMEOUT:
+        unsigned int timeout_new = 0;
+        memcpy(&timeout_new, &packet->data[IN_DATA_BYTE], sizeof(unsigned int));
+        status = set_uhf_watchdog_delay(timeout_new);
```

Comment:
Where is this function defined?

Reply:
Oh yeah I should have noted that this depends on this PR: https://github.com/AlbertaSat/ex2_obc_software/pull/72